docs/pct/README.md

```diff
@@ -0,0 +1,302 @@
+# Puppet Content Templates
+
+* [Overview](#overview)
+* [Getting Started](#getting-started)
+  * [pct new](#pct-new)
+  * [Template Updates](#template-updates)
+  * [More Templates](#more-templates)
+* [Writing Your Own Templates](#writing-templates)
+  * [Dos and Don'ts](#dos-and-donts)
+  * [Structure](#structure)
+    * [pct-config.yml](#pct-configyml)
+  * [Templating Language](#templating-language)
+  * [Testing](#testing)
+  * [Sharing](#sharing)
+* [Overriding Template Defaults](#overriding-template-defaults)
+
+## Overview
+
+Puppet Content Templates (PCT) codify a structure to produce content for any Puppet Product that can be authored by Puppet Product Teams or external users without direct help of the PDK team.
+
+PCT can create any type of a Puppet Product project: Puppet control repo, Puppet Module, Bolt project, etc. It can create one or more independent files, such as CI files or gitignores. This can be as simple as a name for a Puppet Class, a set of CI files to add to a Puppet Module, or as complex as a complete Puppet Control repo with roles and profiles.
+
+These are meant to be ready-to-run, which means they put everything needed for a user to run the project from the moment after creation. This solves the 'blank page' problem, where a few files are in place but the user does not know what the next steps are.
+
+> :warning: PCT is currently in an EXPERIMENTAL phase and feedback is encouraged via [pdkgo/discussions](https://github.com/puppetlabs/pdkgo/discussions) and starting a `feedback` post.
+
+## Getting Started
+
+Install the `exprimental` PCT release available from [github.com/puppetlabs/pdkgo/releases/](https://github.com/puppetlabs/pdkgo/releases/)
+
+Templates currently come in 2 flavours: `project` and `item`.
+
+* A `project` is a template containing many files in a particular structure. They create a ready-to-run structures to start using a Puppet product. _These are great starting points._ You can create a boilerplate empty starter Puppet Module or a fully customized Puppet Module with specialized CI files and RSAPI providers.
+* An `item` is a template that will supplement a project or existing content. These could be language features like a Puppet class or single files like a Git ignore file.
+
+### pct new
+
+PCT is available through the `pct new` command.
+
+The `--list` or `-l` flag displays a list of locally available templates located in the default template location `$HOME/.pdk/pct/`. The list of templates is also available by calling `pct new` without flags.
```

Comment:
**Placeholder Note:** Based on the decision made [here](https://github.com/puppetlabs/pdkgo/issues/20#issuecomment-841178719), we should update the docs, accordingly

Reply:
In progress